lettermgmt/README.rst

```diff
@@ -1,35 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+   
 Letter Management
 =================
 
 Using this module you can track Incoming / Outgoing letters, parcels,
 registered documents or any other paper documents that are important for your
 company to keep track of.
 
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/111/9.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/crm/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/crm/issues/new?body=module:%20lettermgmt%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
 Credits
 =======
 
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
 Contributors
 ------------
 
+* Hardikgiri Goswami <hardikgiri.goswami@gmail.com>
```

Comment:
@rafaelbn I think it's more a tradition than a requirement. In any case, it's done.

@HardikG Thanks for doing it, but when you fix a comment in a way that makes the comment itself disappear in the GitHub conversation (such as this, because you changed the line where the comment was), you don't need to tell you did it; we see it automatically 😉 

Still waiting for the instructions 😊